extensions/apps/world-builder/src/ui/circular-progress.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
```

Comment:
why is this component added here?

Reply:
i thought you need this in the world builder for descriptions no ? previously this was called editor meter